sql/mob_spawn_mods.sql

```diff
@@ -232,6 +232,8 @@ INSERT INTO `mob_spawn_mods` VALUES (16916817,49,1,1);
 
 INSERT INTO `mob_spawn_mods` VALUES (17326081,50,25,1); -- Angra_Mainyu chainspell at 25%
 
+INSERT INTO `mob_spawn_mods` VALUES (17428812,17,1,1); -- Habetrot's Rumble Crawler

```

Comment:
No need to as `no_despawn` mob, all NMs have this by default. But I do think you should add min / max gil mods (2,000~5,625 gil)